backend/de.metas.printing/de.metas.printing.base/src/main/sql/postgresql/system/80-de.metas.printing/5620685_sys_gh12221_UpdateProcesName.sql

```diff
@@ -0,0 +1,159 @@
+;
+
+-- 2022-01-07T09:10:31.274948Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para_Trl SET Help='Product Categories parameter is used to filter the invoice candidates that will be invoiced. Will be filtered out the one that do ot have the product category given as parameter.',Updated=TO_TIMESTAMP('2022-01-07 11:10:31','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='en_US' AND AD_Process_Para_ID=542166
+;
+
+-- 2022-01-07T09:10:55.521742100Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para_Trl SET Help='Product Categories parameter is used to filter the invoice candidates that will be invoiced. Will be filtered out the ones that do not have the product category given as a parameter.',Updated=TO_TIMESTAMP('2022-01-07 11:10:55','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='en_US' AND AD_Process_Para_ID=542166
+;
+
+-- 2022-01-07T09:11:06.844246100Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para_Trl SET Help='Der Parameter Produktkategorien wird verwendet, um die Rechnungskandidaten zu filtern, die in Rechnung gestellt werden. Es werden diejenigen herausgefiltert, bei denen die Produktkategorie nicht als Parameter angegeben ist.',Updated=TO_TIMESTAMP('2022-01-07 11:11:06','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Process_Para_ID=542166
+;
+
+-- 2022-01-07T09:11:11.289021300Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para_Trl SET IsTranslated='Y',Updated=TO_TIMESTAMP('2022-01-07 11:11:11','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Process_Para_ID=542166
+;
+
+-- 2022-01-07T09:11:17.970650100Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para_Trl SET Help='Der Parameter Produktkategorien wird verwendet, um die Rechnungskandidaten zu filtern, die in Rechnung gestellt werden. Es werden diejenigen herausgefiltert, bei denen die Produktkategorie nicht als Parameter angegeben ist.', IsTranslated='Y',Updated=TO_TIMESTAMP('2022-01-07 11:11:17','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_DE' AND AD_Process_Para_ID=542166
+;
+
+-- 2022-01-07T09:11:22.164906Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process_Para SET Help='Der Parameter Produktkategorien wird verwendet, um die Rechnungskandidaten zu filtern, die in Rechnung gestellt werden. Es werden diejenigen herausgefiltert, bei denen die Produktkategorie nicht als Parameter angegeben ist.',Updated=TO_TIMESTAMP('2022-01-07 11:11:22','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Process_Para_ID=542166
```

Comment:
```suggestion
UPDATE AD_Process_Para SET Help='Der Parameter Produktkategorien wird verwendet, um die Rechnungskandidaten zu selektieren, die in Rechnung gestellt werden. Nur Kandidate mit passender Kategorier oder Eltern-Kategorie werden selektiert.', Updated=TO_TIMESTAMP('2022-01-07 11:11:22','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Process_Para_ID=542166
```

..can be done later: the old text hat a doulble negation and was amigous on whether matching ICs are included or excluded.